TumbleBitSetup/PoupardStern/PoupardSternSetup.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TumbleBitSetup
+{
+    public class PoupardSternSetup
+    {
+        public PoupardSternSetup()
+        {
+
+        }
+        public PoupardSternSetup(byte[] publicString, int keySize, int k = 128)
+        {
+            if(publicString == null)
+                throw new ArgumentNullException(nameof(publicString));
+            if(KeySize < 0)
+                throw new ArgumentOutOfRangeException(nameof(keySize));
+            K = k;
+            PublicString = publicString;
+            KeySize = keySize;
+        }
+        public byte[] PublicString
+        {
+            get; set;
+        }
+        public int K
```

Comment:
We need to change the name of this variable, since in my writeup and in the poupard stern paper, capital K (bigK) is the number of RSA values sent with the proof.    This K in setup is the security parameter.

I think the best thing is to call it something like "securityParam" for security parameter, rather than just call it by a letter.  This is much more descriptive anyway and devs mostly know what a security parameter is (I think :) )